README.md

```diff
@@ -66,6 +70,39 @@ A `bfloat16` number is a single-precision float truncated at 16 bits.
 
 Exponent: 8, Mantissa: 7, exponent bias: 127. IEEE 754, with NaN and inf.
 
+### `float4_e2m1`
+
+Exponent: 2, Mantissa: 1, bias: 1.
+
+Extended range: no inf, no NaN.
+
+Microscaling format, 4 bits (encoding: `0bSEEM`) using byte storage (higher 4
+bits are unused). NaN representation is undefined.
+
+Possible values: [0, 0.5, 1, 1.5, 2, 3, 4, 6]
```

Comment:
I'd probably stick backticks around the values

Reply:
What about the negative values?

Reply:
Added backticks around the values (here and below).

Changed to "Possible absolute values" to keep the list short.